src/main/kotlin/com/hibob/kotlinBascis/StoreService.kt

```diff
@@ -0,0 +1,78 @@
+package com.hibob.kotlinBascis
+import org.springframework.stereotype.Component
+import java.time.LocalDate
+
+@Component
+class StoreService {
+
+    fun pay(cart: List<Cart>, payment: Payment): Map<String, Check> {
+        val ans = mutableMapOf<String, Check>()
+        cart.forEach{myCart->
+            val check=checkout(myCart, payment)
```

Comment:
please use CMD + OPTION + L this going to help you align the style in this case it should be:
           ` val check = checkout(myCart, payment)`